### DISCUSSION TITLE
Will PITR be offered?

### DISCUSSION THREAD

**Discussion by puchesjr** [planetscale/discussion#42] [Feature Requests] [answered]
Will point-in-time recovery ("PITR") be offered in the ui?

**Comment by Anders-PlanetScale** [comment]
Hello @puchesjr,

Backup and restore point in time recovery will be available via UI and CLI when PlanetScale DB moves into GA, if not beforehand.

**Reply by puchesjr** [reply]
Can the mysqlbinlog be used currently to perform pitr?

**Reply by pbitkowski** [reply]
@Anders-PlanetScale I saw that you announced that PlanetScale is GA (https://planetscale.com/blog/ga) but still can't see PITR option in the platform. Could you share with us if you have it on your roadmap? Thanks! :)

**Reply by DigitalGreyHat** [reply]
Any updates on this? @jgreet

**Reply by jgreet** [reply]
PITR is still on the roadmap but we don't have an ETA to share at this time.

**Reply by edelbalso** [reply]
Any updates now? @jgreet

**Reply by jgreet** [reply]
No, nothing to announce yet.

**Reply by jessehanley** [reply]
Hey @jgreet any updates here?

**Comment by mscoutermarsh** [comment, marked as answer]
Updating this thread since it's quite out of date. For PITR:

- Postgres: PITR is available. See docs here: https://planetscale.com/docs/postgres/backups/point-in-time-recovery
- Vitess: PITR is currently not available. See docs on backups here: https://planetscale.com/docs/vitess/backups
